plugin/evm/epochs_cache.go

```diff
@@ -0,0 +1,52 @@
+// (c) 2019-2021, Ava Labs, Inc. All rights reserved.
```

Comment:
These files do not seem related to adding unit tests to validators. Did some other changes get merged into this branch?

Reply:
It seems to me that this came from the rebase, do you mean it should not be here?

Reply:
Yes, the rebase might have failed or might need to be done again? Because this PR should only include your commits.